src/Docker.PowerShell/Docker.psm1

```diff
@@ -6,6 +6,7 @@
 # the module is in.
 #
 
+$paramBackup = $PSDefaultParameterValues.Clone()
```

Comment:
Actually, each module gets its own copy of `PSDefaultParameterValues`, so clearing is a nop. You can just delete the line of code and not worry about it.